operate/webapp/src/main/java/io/camunda/operate/webapp/transform/DataAggregator.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.webapp.transform;
+
+/*
+ * Copyright Camunda Services GmbH
```

Comment:
I think this might be the old license header, you might want to check to see if this is supposed to be in here.